src/Sulu/Bundle/AdminBundle/Command/UpdateBuildCommand.php

```diff
@@ -273,16 +273,22 @@ private function doManualBuild(SymfonyStyle $ui): int
         $ui->section('Cleanup previously installed "node_modules" folders');
         $this->cleanupPreviouslyInstalledDependencies();
 
+        $errorHelpMessage =
+            'Visit https://docs.sulu.io/en/latest/cookbook/build-admin-frontend.html#common-errors' . \PHP_EOL
+            . 'to get some tips on how to fix this error.';
+
         $ui->section('Install npm dependencies');
         if ($this->runProcess($ui, 'npm install')) {
             $ui->error('Unexpected error while installing npm dependencies.');
+            $ui->note($errorHelpMessage);
```

Comment:
i think we have not used `note` in the command before. i guess i would use `info` because of that 🙂

Reply:
do you have an idea why why use `$output->writeln`/`$ui->writeln` in some places but `$ui->error` in others? 🙈

Reply:
```suggestion
            $ui->info($errorHelpMessage);
```

Reply:
I don't know why sometimes `output->writeln`/`$ui->writeln` has been used. Maybe it's because of better control of spacing?